lib/oneview-sdk/resource/api300/c7000/enclosure_group.rb

```diff
@@ -26,11 +26,64 @@ def initialize(client, params = {}, api_ver = nil)
           # Default values:
           @data['type'] ||= 'EnclosureGroupV300'
           @data['stackingMode'] ||= 'Enclosure'
+          @data['enclosureCount'] ||= 1
           @data['interconnectBayMappingCount'] ||= 8
-          create_interconnect_bay_mapping unless @data['interconnectBayMappings']
```

Comment:
It does create it, since it calls super. See [here](https://github.com/HewlettPackard/oneview-sdk-ruby/blob/a9a639ca5adb08d17258ab4ac8a35ddd0a4df122/lib/oneview-sdk/resource/api200/enclosure_group.rb#L29).